quint/src/cliCommands.ts

```diff
@@ -558,10 +563,22 @@ export async function verifySpec(prev: TypecheckedStage): Promise<CLIProcedure<V
   return verify(config).then(res =>
     res
       .map(_ => ({ ...verifying, status: 'ok', errors: [] } as VerifiedStage))
-      .mapLeft(err => ({
-        msg: err.explanation,
-        stage: { ...verifying, status: 'failure', errors: err.errors },
-      }))
+      .mapLeft(err => {
+        const trace: QuintEx[] | undefined = err.traces ? ofItf(err.traces[0]) : undefined
+        if (trace !== undefined) {
+          // Always print the conterexample, unless the output is being directed to one of the outfiles
+          const verbosityLevel = prev.args.out || prev.args.outItf ? 0 : 2
```

Comment:
Meaning of the magic constants 0 and 2 is not clear to me.
Should we have names for these?

Reply:
Probably! This mysterious numberings is pervasive throughout this file unfortunately. I’ll make a followup to note the need to apply some cleanup to the verbosity system.

Reply:
Created an issue to follow up! https://github.com/informalsystems/quint/issues/915

Reply:
We should use `defaultLevel` of `verbosity.ts`